tss-esapi/src/context/tpm_commands/non_volatile_storage.rs

```diff
@@ -157,7 +158,35 @@ impl Context {
         }
     }
 
-    // Missing function: NV_Increment
+    /// Increment monotonic counter index
+    ///
+    /// # Details
+    /// This method is used to increment monotonic counter
+    /// in the TPM.
+    pub fn nv_increment(
+        &mut self,
+        auth_handle: NvAuth,
+        nv_index_handle: NvIndexHandle,
+    ) -> Result<()> {
+        let ret = unsafe {
+            Esys_NV_Increment(
+                self.mut_context(),
+                AuthHandle::from(auth_handle).into(),
+                nv_index_handle.into(),
+                self.optional_session_1(),
```

Comment:
I think (?) this should be `required_session_1`, but I think we're very inconsistent in using it, and I'm not sure, at this point, what benefit it brings once we move to the new error code system in v8.0.0. Perhaps this should be something we need to scrap in that release.
(Sorry for the spam, it's not really something required for this PR, just caught sight of it now).